src/models/Trip.js

```diff
@@ -0,0 +1,46 @@
+
+export default (sequelize, DataTypes) => {
+  const Trips = sequelize.define('Trips', {
+    tripId: {
+      type: DataTypes.STRING,
+      required: true
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      required: true
+    },
+    departureDate: {
+      type: DataTypes.STRING,
+      required: true
+    },
+    returnDate: {
+      type: DataTypes.STRING,
```

Comment:
The type for `departureDate` and `returnDate` here should be `DATE` not `STRING`

Reply:
I have implemented this feedback